sdk/cosmos/azure-cosmos/azure/cosmos/exceptions.py

```diff
@@ -66,8 +66,28 @@ class CosmosAccessConditionFailedError(CosmosHttpResponseError):
 
 
 class CosmosBatchOperationError(HttpResponseError):
-    """A transactional batch request to the Azure Cosmos database service has failed."""
-
+    """A transactional batch request to the Azure Cosmos database service has failed.
```

Comment:
This looks great - thanks!
<img width="561" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/8689453/842cd0d8-620b-459c-9428-a0bcea7f2f62">